docs/Developers/Linter_Bears_Advanced.rst

```diff
@@ -41,6 +40,30 @@ The path of the temporary configuration file can be accessed inside
         def create_arguments(filename, file, config_file):
             return "--use-config", config_file
 
+If you return ``None``, no configuration file is generated. A common case
+where to explicitly return ``None`` is when you want to expose a setting
+for the user to use his own tool-specific config. In case the user specifies
+such a config file, we can avoid generating one again with ``generate_config``
+to reduce I/O load.
+
+::
+
+    @linter(executable='...')
+    class MyBear:
+        @staticmethod
+        def generate_config(filename, file,
+                            user_config: str = '',
+                            setting_a: str = ''):
+            if user_config:
+                return 'A=True'
+            else:
+                return None
+
+        def create_arguments(filename, file, config_file,
+                            user_config: str = ''):
+            return (filename, '--use-config', user_config
+                    if user_config else config_file)
```

Comment:
see previous comment:
```
return (filename, '--use-config',
        user_config if user_config else config_file)
```

Reply:
(There's actually a short form, `user_config or config_file`, but let's keep the `if` construct, that's easier to understand)

Reply:
Oh sorry I forgot about that.